assets/cluster-bootstrap/cluster-ingresscontrollers-02-config.yaml

```diff
@@ -4,11 +4,21 @@ metadata:
   name: default
   namespace: openshift-ingress-operator
 spec:
-{{ if .EndpointPublishingStrategyScope }}
-  endpointPublishingStrategy:
-    loadBalancer:
-      scope: {{ .EndpointPublishingStrategyScope }}
-    type: LoadBalancerService
+{{ if or .EndpointPublishingStrategy.Node .EndpointPublishingStrategy.Scope .EndpointPublishingStrategyScope }}
```

Comment:
Now I'm even more confused on what you want to do.  Are you just in need of a nodePort option?  I would like us to take a step back and design this first.

Reply:
Thanks for sending me more background on your use case.  If I understand correctly, you simply want an option to select `NodePortService` versus `LoadBalancerService` for `endpointPublishingStrategy`.  For backwards compatibility purposes, we need to keep `EndpointPublishingStrategyScope` as-is.  Then to do as you want, we could add support for a new `EndpointPublishingStrategy` structure.  At a minimum, the structure would have a `Type` field that could be set to `NodePortService` or `LoadBalancerService` and we could provide default values for the rest of the required values.